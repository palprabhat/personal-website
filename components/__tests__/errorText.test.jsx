import { render, screen } from "@testing-library/react";
import ErrorText from "../errorText";
import { errorTextTest } from "../../constants/testIds.const";

describe("ErrorText Component", () => {
  it("renders correct content", () => {
    const errorMessage = "Test Error message";
    render(<ErrorText testId={errorTextTest}>{errorMessage}</ErrorText>);

    const link = screen.queryByTestId(errorTextTest);

    expect(link).toBeInTheDocument();
    expect(link.innerHTML).toBe(errorMessage);
  });
});
