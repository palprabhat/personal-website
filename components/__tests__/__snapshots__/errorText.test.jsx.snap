// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ErrorText Component renders correct content 1`] = `
<body>
  <div>
    <div
      class="text-red-600 text-sm text-left pl-5 h-6"
    >
      Test Error message
    </div>
  </div>
</body>
`;
